templates/patterns/timeline/pattern-timeline.html.twig

```diff
@@ -8,13 +8,13 @@
 {% set _toggle_collapsed = toggle_collapsed|default('Show more %d items'|t) %}
 {% set _toggle_expanded = toggle_expanded|default('Hide %d items'|t) %}
 {% include '@oe-bcl/timeline' with {
-  toggle_collapsed: _toggle_collapsed,
-  toggle_expanded: _toggle_expanded,
-  hide: {
-    from: hide_from,
-    to: hide_to|default(0),
+  'toggle_collapsed': _toggle_collapsed,
+  'toggle_expanded': _toggle_expanded,
```

Comment:
```suggestion
  'toggle_collapsed': toggle_collapsed|default('Show more %d items'|t) ,
  'toggle_expanded': toggle_expanded|default('Hide %d items'|t),
```
By the way, we need to add "label" to the name, or it seems a boolean variable. "collapse_button_label" and "expand_button_label" i think they are better.